climada/util/coordinates.py

```diff
@@ -85,6 +85,79 @@
 are not considered."""
 
 
+def check_if_geo_coords(lat, lon):
+    """Check if latitude and longitude arrays are likely in geographic coordinates,
+    testing if min/max values are within -90 to 90 for latitude and -540 to 540
+    for longitude. Lon coordinates of <-360 or >360 are allowed to cover cases
+    of objects being defined close to the 180 meridian for which longitude
+    intervals such as [179, 538] or [-538, -179] might be used.
+
+    Parameters
+    ----------
+    lat, lon : ndarrays of floats, same shape
+        Latitudes and longitudes of points.
+
+    Raises
+    ------
+    ValueError : if lat or lon out of likely geographical coordinate bounds
+
+    Returns
+    -------
+    test : bool
+        True if lat/lon ranges seem to be in the geographic coordinates range, otherwise False.
```

Comment:
This function never returns `False`. It either returns `True` or raises an error. I think you should split that functionality: Have a function `is_geo_coords` (or similar) that returns true/false, and then have a function `check_XZY`, that raises an error or returns None.

Reply:
Done.